src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -48,645 +48,647 @@
                             </div>
-                        </div>
-                        <div class="form-group">
-                            {{ form_widget(form.step2.on_sale) }}
-                        </div>
-                        <div class="alert alert-info" role="alert">
-                            {{ trans('Final retail price : ', {}, 'AdminProducts') }}
-                            <strong><span id="final_retail_price_ti"></span> {{ trans('tax incl.', [], 'AdminProducts') }}</strong> /
-                            <span id="final_retail_price_te"></span> {{ trans('tax excl.', [], 'AdminProducts') }}
-                        </div>
 
-                        <div class="form-group">
-                            <h4>{{ trans('Cost price', {}, 'AdminProducts') }}</h4>
-                            <div class="col-xs-3 pull-left">
-                                <label class="control-label">{{ form.step2.wholesale_price.vars.label|raw }}</label>
-                                {{ form_errors(form.step2.wholesale_price) }}
-                                {{ form_widget(form.step2.wholesale_price) }}
+                            <div class="form-group">
+                                <h4>{{ trans('Cost price', {}, 'AdminProducts') }}</h4>
+                                <div class="col-xs-3 pull-left">
+                                    <label class="control-label">{{ form.step2.wholesale_price.vars.label|raw }}</label>
+                                    {{ form_errors(form.step2.wholesale_price) }}
+                                    {{ form_widget(form.step2.wholesale_price) }}
+                                </div>
                             </div>
-                        </div>
 
-                        <div class="form-group" id="specific-price">
-                            <h4>{{ trans('Specific price', {}, 'AdminProducts') }}</h4>
-
-                            <div class="col-sm-10">
-                                <table id="js-specific-price-list" class="table table-striped hide" data="{{ path('admin_specific_price_list') }}" data-action-delete="{{ path('admin_delete_specific_price') }}">
-                                    <thead>
-                                    <tr>
-                                        <th>{{ trans('Rule', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Combination', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Currency', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Country', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Group', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Customer', {}, 'AdminProducts') }}</th>
-                                        <th>{{ trans('Fixed price', {}, 'A
```

Comment:
would you mind to move it into his own template ? A simple copy paste for you, but better readibility for any front end developper ;)